README.md

```diff
@@ -16,6 +16,40 @@ minimal requirements with respect to application structure, routes etc. With
 its pluggable strategies it __can support all kinds of authentication and
 authorization mechanisms__.
 
+# Table of Contents
+
+**Information**
+
+* [What does it do?](#what-does-it-do)
+* [How does it work?](#how-does-it-work)
+* [Working example](#how-do-i-use-it)
+
+**Usage**
+
+* [Installation](#installation)
+* [Basic Usage](#basic-usage)
+
+**Core Building Blocks**
```

Comment:
Maybe we should rename this to _"Guide"_ or so as it actually talks about testing, implementing custom authenticators etc. as well?